humble/distribution.yaml

```diff
@@ -1716,6 +1716,16 @@ repositories:
       url: https://github.com/ros-drivers/gscam.git
       version: ros2
     status: developed
+  gtsam:
+    doc:
+      type: git
+      url: https://github.com/borglab/gtsam/
```

Comment:
These are git repository URLs and should end with `.git` to avoid an extra redirect. That's what CI is complaining about. Same change to the other 3 instances of this URL.
```suggestion
      url: https://github.com/borglab/gtsam.git
```

Reply:
Thanks!